packages/components/combobox/src/grouped-option.ts

```diff
@@ -0,0 +1,30 @@
+import { Option } from '@sl-design-system/listbox';
+import { type CSSResultGroup, type TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import styles from './grouped-option.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-combobox-grouped-option': GroupedOption;
+  }
+}
+
+export class GroupedOption extends Option {
```

Comment:
Can you please also add a small description above just like we have in other components? :)